src/js/constants.js

```diff
@@ -83,6 +83,7 @@ export const MediaUIAttributes = {
   MEDIA_LOADING: 'media-loading',
   MEDIA_BUFFERED: 'media-buffered',
   MEDIA_STREAM_TYPE: 'media-stream-type',
+  MEDIA_CONTAINER_SIZE: 'media-container-size'
```

Comment:
question(blocking): Is this a `MediaUIAttribute`? It isn't modeling media state and it isn't designed to be shared with Media State Receivers (aka other Media Chrome Components). I think don't want to muddy that distinction here.

Reply:
good point, will change it

Reply:
though, it is state derived from media-controller which is influenced by the media.

there's also `MediaStateReceiverAttributes`

```js
export const MediaStateReceiverAttributes = {
  MEDIA_CHROME_ATTRIBUTES: 'media-chrome-attributes',
  MEDIA_CONTROLLER: 'media-controller',
};
```

is it better this to be `containersize`? then it almost feels like one of the attributes the user controls

Reply:
ah no that's a good point... I think the biggest concern is adding it to the `MediaUIAttribute` object, which is exclusively for state propagation stuff. I think we can keep the name but it probably should be in a different/new object (or just exported as its own const? defer to you on where it should be moved).